docs/de/configuration/codelists/README.md

```diff
@@ -0,0 +1,39 @@
+# Codelisten
+
+Codelisten können zum Übersetzen von Eigenschaftswerten in einen anderen Wert genutzt werden, meist für die HTML-Ausgabe. Die Codelisten müssen als JSON-Dateien im ldproxy-Datenverzeichnis unter dem relativen Pfad `store/entities/codelists/{codelistId}` liegen.
```

Comment:
In Configuration you say that all files in the store are YAML. I don't think there is technical reason to use JSON here. 
Or did you try it and it did not work? I just saw that you also use JSON files without extensions for the codelists in inspire-nrw. That is supported only for backwards compatibility. Files ending with `.json`, `.yaml` or `.yml` should be parsed accordingly.

Reply:
Somehow I never tried to change the codelists to yaml... I have converted my local codelists now and everything works fine. I have updated the text.